test/mocks/event/mocks.cc

```diff
@@ -51,6 +51,19 @@ MockTimer::MockTimer(MockDispatcher* dispatcher) : MockTimer() {
 
 MockTimer::~MockTimer() = default;
 
+MockRangeTimer::MockRangeTimer() {
```

Comment:
Were these changes intended to be part of this PR or some future PR?

Reply:
This one. You mentioned expecting a mock, I assumed it should be included here. I can hold off on this until later though.